app/src/main/java/com/claraberriel/mvpparking/mvp/view/ReservationView.java

```diff
@@ -0,0 +1,119 @@
+package com.claraberriel.mvpparking.mvp.view;
+
+import android.content.Context;
+import android.widget.Toast;
+
+import androidx.fragment.app.Fragment;
+
+import com.claraberriel.mvpparking.R;
+import com.claraberriel.mvpparking.databinding.FragmentReservationBinding;
+import com.claraberriel.mvpparking.utilities.Picker;
+
+import java.util.Date;
+
+public class ReservationView extends FragmentView {
+
+    private final FragmentReservationBinding reservationBinding;
+    private final Context context;
+    private final Picker startDatePicker;
+    private final Picker endDatePicker;
+    
+    public ReservationView(Fragment fragmentRef, FragmentReservationBinding reservationBinding) {
+        super(fragmentRef);
+        this.reservationBinding = reservationBinding;
+        this.context = getContext();
+        startDatePicker = new Picker(reservationBinding.startDateTime);
+        endDatePicker = new Picker(reservationBinding.endDateTime);
+    }
+
+    /**
+     * Show Date and Time Dialogs
+     */
+
+    public void startDateTimeDialog() {
+        startDatePicker.show();
+    }
+
+    public void endDateTimeDialog() {
+        endDatePicker.show();
+    }
+
+    /**
+     * Getters
+     * @return values from EditTexts inputs
+     */
+
+    public Date getStartDate() {
+        return startDatePicker != null ? startDatePicker.getDate() : null;
+    }
+
+    public Date getEndDate(){
+        return endDatePicker != null ? endDatePicker.getDate() : null;
+    }
+
+    public String getParkingNumber() throws NumberFormatException {
```

Comment:
This won't throw NumberFormatException, so there is no need to declare it on the method signature.